hack/make-rules/verify.sh

```diff
@@ -21,6 +21,15 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
+# include shell2junit library
```

Comment:
When would this be true? Isn't it checked in to the repo as well?

Reply:
It is. I don't know why this wouldn't trigger, but I had copied it from here: https://github.com/kubernetes/kubernetes/blob/884e4b7b9c63a9a3b94278972d640454b8d16e47/hack/make-rules/test-cmd-util.sh#L82-L89

Reply:
Maybe we can drop it then, seems like a bit of a cargo cult situation :D